libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Instrumentation/SFSDKInstrumentationHelper.h

```diff
@@ -30,6 +30,8 @@
 NS_ASSUME_NONNULL_BEGIN
 
 NS_SWIFT_NAME(SalesforceInstrumentationHelper)
+/// @deprecated Signpost logging is deprecated and this helper will be removed in version 14.0.
+__attribute__((deprecated("Signpost logging is deprecated and will be removed in version 14.0")));
```

Comment:
Is it possible to use the [SFSDK_DEPRECATED macro](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/ee533a680cb73fa02a9c0f77dd57618ea08fcedf/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKConstants.h#L105) instead?

Reply:
yes